src/exposurelog/resolvers/find_messages.py

```diff
@@ -16,8 +16,8 @@
 async def find_messages(
     app: aiohttp.web.Application,
     _info: graphql.GraphQLResolveInfo,
-    **kwargs,
-) -> dict:
+    **kwargs: typing.Any,
+) -> typing.List[dict]:
```

Comment:
Again, I thought you could just use `list[dict]`

Reply:
I have changed it everywhere.